src/containers/Game/Game.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react';
+
+import classes from './Game.scss';
+import { Slots } from '../../components/Slots/Slots';
+import { Letters } from '../../components/Letters/Letters';
+
+import { words } from '../../data/data';
+
+const generateID = () => `_${Math.random().toString(36).substr(2)}`;
+
+words.map((el) => (
+    {
+        ...el,
+        id: generateID(),
+    }
+));
+
+class Game extends Component {
+    state = {
+        wordsNumber: words.length,
+        word: {},
+        wordLetters: [],
+        shuffledLetters: [],
+        lettersInSlots: [],
+        // eslint-disable-next-line
```

Comment:
It's made for future reference. I wanted the `gameOver` function to be complete.

but I will be more specific about a rule I disable.

Reply:
let me add some reasons:
1. keeping code changes related to certain feature in one commit or one pull request gives cleaner git/github history; in next pull request, when you work on game completion mechanics it would be nice to see all related changes 
2. you may easily forget about the existence of such variables leaving unused variables

Reply:
👍